src/Phinx/Migration/Manager/Environment.php

```diff
@@ -231,7 +231,8 @@ public function getVersions()
     }
 
     /**
-     * Get all migration log entries, indexed by version number.
+     * Get all migration log entries, indexed by version name and sorted ascendingly by the configuration's version 
+     * order option
```

Comment:
right, this was the "english name" of the configuration option, wasn't meant to be taken literally, but to avoid confusion I'll rename it to the actual configuration option